app/views/layouts/_support_header.html.erb

```diff
@@ -6,4 +6,5 @@
   <%= nav_link 'Features', support_interface_feature_flags_path, active: 'feature_flags' %>
   <%= nav_link 'Performance', support_interface_performance_path, active: 'performance' %>
   <%= nav_link 'Tasks', support_interface_tasks_path, active: 'tasks' %>
+  <%= nav_link 'Support users', support_interface_support_users_path, active: 'users' %>
```

Comment:
This is wrapping now.

I wonder if we should move some of these behind an "Admin" link?

Admin page would link to:
- Features
- Tasks
- Performance
- Support users

(probably not in this PR)

Reply:
I discussed with @tvararu — we'd like to revisit this. Will loop you in @fofr when that conversation happens

Reply:
Actually Paul's suggestion sounds good, it was what I was thinking about too 🤔 if it's not too much, we can do it as part of this?

Reply:
I'll do another PR after this